rudder-lang/src/rudderlang_lib.rs

```diff
@@ -0,0 +1,243 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
created a rudderlanglib that parses the rudderlang lib and some utility checks for Technique + a libmethod that helps checking